Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/openshift/origin-must-gather:4.3.0 as builder
+FROM quay.io/openshift/origin-must-gather:4.6.0 as builder
```

Comment:
This would copy /usr/bin/oc from rhel8 to centos7, are we sure it's ok ?

Reply:
I think the change here is just changing which version gets copied

Reply:
> are we sure it's ok

"it worked on my machine..." :slightly_smiling_face:

Reply:
we need to make this change, but can it be a separate patch that moves both the builder and the image to el8 ?

Reply:
removed this change